src/WinRT.Runtime/ComWrappersSupport.cs

```diff
@@ -343,12 +343,13 @@ private static bool ShouldProvideIReference(Type type)
         {
             static bool IsWindowsRuntimeType(Type type)
             {
-                if (type.GetCustomAttribute<WindowsRuntimeTypeAttribute>() is object)
+                if ((type.GetCustomAttribute<WindowsRuntimeTypeAttribute>() is object) ||
+                    WinRT.Projections.IsTypeWindowsRuntimeType(type))
```

Comment:
Is it worth it to make a local var for `WinRT.Projections.IsTypeWindowsRuntimeType(type)` since we use it here and at the exit statement (L352)

Reply:
type gets reassigned between calls